README.md

```diff
@@ -12,52 +12,53 @@ You can protect the DM area by setting a password.
 
 ![alt text](https://apclary.github.io/dungeon-revealer/img/example_player_1.jpeg "Player's view")
 
-## Installation
+## Getting Started
 
-### Docker
+### Getting the app
 
-A docker image is kept up to date with master branch of this repository.
+The easiest way to use dungeon-revealer is to download the app from the [releases](https://github.com/apclary/dungeon-revealer/releases) page here on github. There is also a [docker image](https://hub.docker.com/r/apclary/dungeon-revealer) that is kept up to date with the releases in this repository.
 
-To create an instance run the following:
+#### Prebuilt app 
 
-`docker run -e DM_PASSWORD=<password> -p <PORT>:3000 -d apclary/dungeon-revealer`
+Download the app for your system from [here](https://github.com/apclary/dungeon-revealer/releases).
```

Comment:
I would say let's make that line bold. MIght also mention that we have binaries for windows, linux and OSX